test/cypress/support/deployments.ts

```diff
@@ -30,6 +30,8 @@ const commands = {
             siteName !== '' ? { site_name: siteName } : { detach_site: true }
         );
     },
+    getReservedLabelKeys: () =>
+        cy.cfyRequest('/labels/deployments?_reserved=true', 'GET', null).then(response => response.body.items),
```

Comment:
If we added a type annotation for `items`, the whole function would be safer :slightly_smiling_face: 

```suggestion
        cy.cfyRequest('/labels/deployments?_reserved=true', 'GET', null).then(response => response.body.items as string[]),
```

Reply:
Done.